### ISSUE TITLE
Jaws tool unable to read the dropdown control  label

### ISSUE THREAD

**Issue by priaajais1** [microsoft/fluentui#16309]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (Chrome,Edge,IE)

### Describe the issue:

Using the Jaws tool if we navigate till the dropdown control it misses the label name and just announces the options. But this issue does not repro with narrator and NVDA

https://user-images.githubusercontent.com/76722999/103263460-f572d900-49cd-11eb-819b-60a743f42f6e.mp4

![image](https://user-images.githubusercontent.com/76722999/103263082-d1fb5e80-49cc-11eb-9e9a-8116dc405f5c.png)

#### Actual behavior:

Does not read the label

#### Expected behavior:
Should read the dropdown label

Fluent URL: https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown

**Comment by gouttierre** [comment]
@priaajais1 - thanks for the feedback.

I could not repo your issue on our latest version  [Fluent dropdown](https://developer.microsoft.com/en-us/fluentui#/controls/web/dropdown). Please can you confirm which version you had this issue with? And if you have tried using our latest release.  Thanks and regards.
